src/tiledimage.js

```diff
@@ -751,6 +793,16 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
+     * @private
+     * Get the point around which this tiled image is rotated
```

Comment:
From what I remember, I made sure that the `@private` comments do not appear in the documentation anymore.
I think this is that line: https://github.com/openseadragon/site-build/blob/master/Gruntfile.js#L110